Doc/library/ftplib.rst

```diff
@@ -17,7 +17,7 @@ This module defines the class :class:`FTP` and a few related items. The
 this to write Python programs that perform a variety of automated FTP jobs, such
 as mirroring other FTP servers.  It is also used by the module
 :mod:`urllib.request` to handle URLs that use FTP.  For more information on FTP
-(File Transfer Protocol), see Internet :rfc:`959`.
+(File Transfer Protocol), see internet :rfc:`959`.
```

Comment:
Not sure because not sure why 'Internet' is present in reference.

Reply:
I was thinking about removing `internet` from this sentence, but the `internet RFC-*` pattern is used few times in docs. I can sent subsequent PR.

Reply:
I think it is an adjective here and it is correct to lowercase it.